### PR TITLE
docs(cloud): add Cube Cloud to list of deployment platforms

### PR DESCRIPTION
**Pull request by keydunov**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

File: docs/content/Deployment/Deployment-Guide.md [hunk 1]
```diff
@@ -411,3 +412,25 @@ git push heroku master
...
+
+## Cube Cloud
+
+[[info |]]
+| [Cube Cloud][link-cube-cloud] currently is in early access. If you don't have
+| an account yet, you can [sign up to the waitlist here][link-cube-cloud].
+
+[link-cube-cloud]: https://cube.dev/cloud
+
+Cube Cloud is purpose-built platform to run Cube.js applications in production.
```
Comment (rpaik) [inline docs/content/Deployment/Deployment-Guide.md:424]:
```suggestion
Cube Cloud is a purpose-built platform to run Cube.js applications in production.
```

File: docs/content/Deployment/Deployment-Guide.md [hunk 2]
```diff
@@ -411,3 +412,25 @@ git push heroku master
...
+It is made by the creators of Cube.js and incorporates all the best practices of
+running and scaling Cube.js applications.
+
+![](https://raw.githubusercontent.com/statsbotco/cube.js/master/docs/content/Developer-Tools/how-cube-cloud-works.png)
+
+Cube Cloud can be integrated with your Github to automtically deploy from the specified production branch (master by default). It also can create staging and preview APIs based on the branches in the repository.
```
Comment (rpaik) [inline docs/content/Deployment/Deployment-Guide.md:430]:
```suggestion
Cube Cloud can be integrated with your GitHub to automatically deploy from the specified production branch (master by default). It also can create staging and preview APIs based on the branches in the repository.
```

File: docs/content/Deployment/Deployment-Guide.md [hunk 3]
```diff
@@ -411,3 +412,25 @@ git push heroku master
...
+| [Cube Cloud][link-cube-cloud] currently is in early access. If you don't have
+| an account yet, you can [sign up to the waitlist here][link-cube-cloud].
+
+[link-cube-cloud]: https://cube.dev/cloud
+
+Cube Cloud is purpose-built platform to run Cube.js applications in production.
+It is made by the creators of Cube.js and incorporates all the best practices of
+running and scaling Cube.js applications.
+
+![](https://raw.githubusercontent.com/statsbotco/cube.js/master/docs/content/Developer-Tools/how-cube-cloud-works.png)
```
Comment (hassankhan) [inline docs/content/Deployment/Deployment-Guide.md:428]:
```suggestion
![](https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Developer-Tools/how-cube-cloud-works.png)
```

File: docs/content/Deployment/Deployment-Guide.md [hunk 4]
```diff
@@ -411,3 +412,25 @@ git push heroku master
 
 For more details, take a look at the
 [official documentation from Heroku](https://devcenter.heroku.com/articles/build-docker-images-heroku-yml).
+
+## Cube Cloud
+
+[[info |]]
+| [Cube Cloud][link-cube-cloud] currently is in early access. If you don't have
+| an account yet, you can [sign up to the waitlist here][link-cube-cloud].
```
Comment (hassankhan) [inline docs/content/Deployment/Deployment-Guide.md:420]:
```suggestion
<!-- prettier-ignore-start -->
[[info |]]
| [Cube Cloud][link-cube-cloud] currently is in early access. If you don't have
| an account yet, you can [sign up to the waitlist here][link-cube-cloud].
 <!-- prettier-ignore-end -->
```

Comment (hassankhan) [reply docs/content/Deployment/Deployment-Guide.md:420]: @keydunov Should the waitlist link be different to the main Cube Cloud one?

**Review by hassankhan — APPROVED** [review]
Minor additions, looks great otherwise.

File: docs/content/Deployment/Deployment-Guide.md (hunk 4 above)
Comment (keydunov) [reply docs/content/Deployment/Deployment-Guide.md:421]: We have only one page so far.

**Comment by keydunov** [comment]
Thank you @hassankhan and @rpaik !

**Pull request merged**